content/en/docs/Getting started/_index.md

```diff
@@ -13,82 +13,132 @@ description: >
 
 ### Install Hugo 
 
-You need a [recent version](https://github.com/gohugoio/hugo/releases) of Hugo to do local builds and previews of sites (like this one) that use Docsy. If you install from the release page, make sure to get the `extended` Hugo version, which supports SCSS: you may need to scroll down the list of releases. Hugo can be installed via Brew if you're running MacOs. If you're a Linux user, do not use `sudo apt-get install hugo`, as it currently doesn't get you the `extended` version.
+You need a [recent version](https://github.com/gohugoio/hugo/releases) (we recommend version 0.54.0 or later) of [Hugo](https://gohugo.io/) to do local builds and previews of sites (like this one) that use Docsy. If you install from the release page, make sure to get the `extended` Hugo version, which supports [SCSS](https://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html); you may need to scroll down the list of releases to see it. 
 
-For comprehensive Hugo documentation, see [gohugo.io](https://gohugo.io/)
+#### Linux
+
+Do **not** use `sudo apt-get install hugo`, as it currently doesn't get you the `extended` version. 
+
+If you've already installed Hugo, check your version:
+
+```
+hugo version
+```
+If the result is `v0.5` or earlier, or if you don't see `Extended`, you'll need to install the latest version.
+    
+1.  Go to the [Hugo releases](https://github.com/gohugoio/hugo/releases) page.
+2.  In the most recent release, scroll down until you find a list of
+    **Extended** versions.
+3.  Download
+    [hugo_extended_0.52_Linux-64bit.tar.gz](https://github.com/gohugoio/hugo/releases/download/v0.52/hugo_extended_0.52_Linux-64bit.tar.gz)
+    (or the latest version).
+4.  Create a new directory:
+
+        mkdir hugo
+
+5.  Extract the files you downloaded to `hugo`.
+
+6.  Switch to your new directory:
+
+         cd hugo
+
+7.  Install Hugo:
+
+        sudo install hugo /usr/bin    
+
+For comprehensive Hugo documentation, see [gohugo.io](https://gohugo.io/).
+
+#### macOS
+
+Install Hugo using [Brew](https://gohugo.io/getting-started/installing/#homebrew-macos).
 
 ### Install PostCSS
 
-If you want to make any stylesheet changes, you will also need `PostCSS` to create the final assets:
+If you plan to make any stylesheet changes, you will also need `PostCSS` to create the final assets:
```

Comment:
I've realized that with the current Netlify workaround of doom you actually need postcss locally or the resources don't build (ditto building the site locally for the first time). For now we should change this to:

To build or update your site's CSS resources, you also need `PostCSS` to create the final assets:

Reply:
done